### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by 2otherjobs**

### CODE REVIEW

File: Backup-zfs-filesystems-with-Znapzend [hunk 1]
```diff
@@ -0,0 +1,246 @@
+ZNAPZEND - BACKUP YOUR DATA USING ZFS' BUILTIN SNAPSHOT AND SEND/RECEIVE FEATURES
```
Comment (Mrt134) [inline Backup-zfs-filesystems-with-Znapzend]:
```suggestion
ZNAPZEND - BACKUP YOUR DATA USING ZFS BUILTIN SNAPSHOT AND SEND/RECEIVE FEATURES
```

**Review by Mrt134 — APPROVED** [review]
I read through the instructions and tested as many commands as I could for a quick sanity check.
Couple of small suggestions:
- The root user is shown doing the commands, but there isn't an instruction or example of switching to the root account. Might be helpful or prevent an error to add a sentence showing the `su` or `sudo` commands.
- There's a couple textual references to outside resources that could be enhanced by adding a link to said resource.
- Could be helpful to add manual page commands if the reader needs to get quick help from the CLI: `man znapzend`, `man znapzendzetup`.
- The `SUMMARY` section would be more helpful at the beginning of the tutorial.

**Comment by Mrt134** [comment]
Let me know if you're ready for the PR to be merged or if you're still doing review/refinement on the text. Thanks for writing and submitting this guide!

**Comment by 2otherjobs** [comment]
Thanks Tim for your excellent suggestions.  I will execute them ASAP, probably later today.

Tom Bartlett
2otherjobs

File: Backup-zfs-filesystems-with-Znapzend [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ZNAPZEND - BACKUP YOUR DATA USING ZFS' BUILTIN SNAPSHOT AND SEND/RECEIVE FEATURES
+ZNAPZEND - BACKUP YOUR DATA USING ZFS BUILTIN SNAPSHOT AND SEND/RECEIVE FEATURES
 
 Znapzend automates BOTH creation of zfs snapshots AND sending them to another filesystem.
 Most other zfs utilities do one or the other but not both.
```
Comment (2otherjobs) [inline Backup-zfs-filesystems-with-Znapzend]: SUMMARY
This tutorial begins with the background steps to enable znapzend to create zfs snapshots 
and then send them to either a local USB drive, another LAN server, or both. A simple yet
complete example for snapshot creation and send/receive configuration is provided.  For 
users just getting started with FreeBSD and Project-Trident, the extra detail may be helpful.

**Comment by 2otherjobs** [comment]
I just finished the next re-edit.  Check it out at your convenience.

Tom Bartlett
2otherjobs

**Review by Mrt134 — APPROVED** [review]
Great, looks good! Thanks for the submission.

**Pull request merged**